src/lib.rs

```diff
@@ -70,6 +70,13 @@ pub fn create_ecx<'a, 'mir: 'a, 'tcx: 'mir>(
         Evaluator::new(validate),
     );
 
+    let stack_addr = ecx.memory_mut().allocate(
+        Size::ZERO,
+        Align::from_bytes(fn_call::EMULATED_PAGE_SIZE.into()).unwrap(),
+        MiriMemoryKind::StackRoot.into(),
+    );
+    ecx.machine.stack_addr = stack_addr.with_default_tag();
```

Comment:
Same here, this needs a comment what this is doing why, and why the size and alignment are the way they are.